protractor/e2e/integration/ng2.spec.ts

```diff
@@ -58,5 +58,13 @@ describe('Main page test suite', () => {
         await secondPage.infoSubmitButton.click();
     });
 
+    it('Check blog page',async () => {
+        await secondPage.blogButton.click();
+        expect(await secondPage.getCurrentUrl()).toContain(secondPage.blogLink);
+        expect(await secondPage.postsList.count()).toEqual(20);
+        expect(secondPage.isElementDisplayed(await secondPage.navigationBar)).toBeTruthy();
+        await secondPage.linkTosSecondPage.click();
+        expect(await secondPage.postsList.count()).toBeTruthy()
+    });
```

Comment:
It's advisable to divide all this logic into 2 it blocks:
The first one for checking "Navigate to blog and check articles, pages navigation"
And the second one: "Navigate to blog and check articles, pages navigation"